### PR TITLE
Fixed error code typo

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs/src/autorest.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e59b6449e6613c0e4ecb73daacacdff320d65f67/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-08-06/blob.json
+    - https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/25157cb0edcbc75b234d08467cfc9bba385b6b0f/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-08-06/blob.json
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Blobs/src/autorest.md:7]: I'm assuming the swagger PR hasn't been merged yet?

File: sdk/storage/Azure.Storage.Blobs/src/Generated/Models/BlobErrorCode.cs [hunk 2]
```diff
@@ -84,7 +84,7 @@ public BlobErrorCode(string value)
         private const string CopyIdMismatchValue = "CopyIdMismatch";
         private const string FeatureVersionMismatchValue = "FeatureVersionMismatch";
         private const string IncrementalCopyBlobMismatchValue = "IncrementalCopyBlobMismatch";
-        private const string IncrementalCopyOfEralierVersionSnapshotNotAllowedValue = "IncrementalCopyOfEralierVersionSnapshotNotAllowed";
+        private const string IncrementalCopyOfEarlierVersionSnapshotNotAllowedValue = "IncrementalCopyOfEarlierVersionSnapshotNotAllowed";
```
Comment (EmmaZhu) [inline sdk/storage/Azure.Storage.Blobs/src/Generated/Models/BlobErrorCode.cs:87]: Does server side also use the same error code?

File: sdk/storage/Azure.Storage.Blobs/src/autorest.md (hunk 1 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs/src/autorest.md]: Yep, fixed.

File: sdk/storage/Azure.Storage.Blobs/src/Generated/Models/BlobErrorCode.cs (hunk 2 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs/src/Generated/Models/BlobErrorCode.cs:87]: Ryan has confirmed that the service is not returning the misspelled error code.

**Pull request merged**